lib/app.js

```diff
@@ -13,7 +13,7 @@
 
 
     function addCalculator() {
-      calculators.push(new Calculator());
+      calculators.push(" :) ");
```

Comment:
now it's working
https://github.com/SMH110/widgets/commit/4c008ecdac1ccfc5387a2c59ce91d0373828651b